// judge_service.proto
syntax = "proto3";

service JudgeService {
    // Method definitions here
    rpc GetFeature(JudgeServiceReq) returns (JudgeServiceRsp) {}
}

message JudgeServiceReq {
    // Define the request field here
    int32 id = 1;
}

message JudgeServiceRsp {
    // Define the response field here
    int32 result = 1;
}